test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -223,6 +223,7 @@ Function sys.babelfish_waitfor_delay(text)
 Function sys.babelfish_waitfor_delay(timestamp without time zone)
 Function sys.bbf_binary_cmp(sys.bbf_binary,sys.bbf_binary)
 Function sys.bbf_get_current_physical_schema_name(text)
+Function sys.bbf_is_shared_schema(text)
```

Comment:
this indicates that we have not added dependency test for this view

Reply:
Fixed